src/lava/magma/runtime/message_infrastructure/message_infrastructure_interface.py

```diff
@@ -13,6 +13,11 @@
 from lava.magma.compiler.channels.interfaces import ChannelType, Channel
 from lava.magma.core.sync.domain import SyncDomain
 
+"""A Message Infrastructure Interface which can create actors which would
```

Comment:
If you capitalize everything, it should probably written as a concatenated CamelCase string.